src/pages/documentation/identity-for-edge-network/faq.md

```diff
@@ -141,3 +141,11 @@ iOS
 ## Q: Can I safely remove the Identity for Experience Cloud ID Service extension in an app if I am using the Edge Network extension?
 
 If no other extension relies on the Identity for Experience Cloud ID Service extension, you can safely remove it. Please refer to the [Identity consideration document](../adobe-analytics/migrate-to-edge-network.md#identity-consideration-for-the-migration) for more information.
+
+## Q: In my hybrid app, I need to pass the mobile SDK's ECID to the WebView using getUrlVariables API. However, the Web SDK generates a new ECID instead of recognizing the existing one. How to fix this?
+
+1. Ensure that the output from the getUrlVariables API includes accurate ECID and OrgID values.
```

Comment:
.. and Experience Cloud orgID values. Please note that the orgID set up for the Mobile SDK must match the orgID configured in the Web SDK.